spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/utils/SparkBaseTableHelper.java

```diff
@@ -42,34 +39,31 @@
  * could implement more capabilities like SupportsPartitionManagement for Hive table, SupportsIndex
  * for JDBC table, SupportsRowLevelOperations for Iceberg table.
  */
-public abstract class SparkBaseTable implements Table, SupportsRead, SupportsWrite {
+public class SparkBaseTableHelper {
```

Comment:
Could you remove `sparkTable` from this helper class? It would be more clear if this class focuses on how to retrieve information from Gravitino, WDYT?  How about rename to  something like `GravitinoTableInfoHelper`?

Reply:
the comment of the class should be changed.

Reply:
> Could you remove `sparkTable` from this helper class? It would be more clear if this class focuses on how to retrieve information from Gravitino, WDYT? How about rename to something like `GravitinoTableInfoHelper`?

@FANNG1 
i would like to use the real `sparkTable` to support Iceberg time travel, so I want to keep `sparkTable` in helper class.
In this context, does it seem that `SparkBaseTableHelper` is more appropriate than `GravitinoTableInfoHelper`?

Reply:
> the comment of the class should be changed.

got it.

Reply:
> > i would like to use the real sparkTable to support Iceberg time travel, so I want to keep sparkTable in helper class.
> 
> could you provide more information why should use `sparkTable` in this helper class?

for Iceberg time travel, the `snapshotId`  of Iceberg `SparkTable` has been initialized  when `loadTable` with `timestamp` or `version`. and then it will be passed to `newScanBuilder` to read the specified snapshot. so i would like to use the real sparkTable in helper class to override the `newScanBuilder` to support Iceberg time travel.
And it seems easier to keep consistent with Hive behavior in helper class.

1. https://github.com/apache/iceberg/blob/2857d3a927527e8116871ef4abe6cfed15340188/spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkCatalog.java#L176-L233

2. https://github.com/apache/iceberg/blob/2857d3a927527e8116871ef4abe6cfed15340188/spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkTable.java#L268-L282

Reply:
cc @FANNG1

Reply:
seems no conflict with move internal spark table (include `newScanBuilder` logic) to  `SparkXXTable`,  the helper class only focus on how to retrieve information from Gravitino.

Reply:
ok

Reply:
updated.